### PR TITLE
Update UE perf recommendations, add a doc for deploy to device, add a note about debugging, add a note about capabilities over streaming, minor fixes

### PR DESCRIPTION
**Pull request by sw5813**

### CODE REVIEW

File: mixed-reality-docs/unreal-streaming.md [hunk 1]
```diff
@@ -27,7 +27,8 @@ To get started, you'll need to download the [Holographic Remoting Player](hologr
 When streaming, you have access to almost all of the same HoloLens capabilities as you would when running an application on a device. This includes [hand joint tracking](unreal-hand-tracking.md) (if you're on a HoloLens 2), [spatial mapping](unreal-spatial-mapping.md), and [spatial anchors](unreal-spatial-anchors.md), but leaves out the features on this [list of limitations](holographic-remoting-troubleshooting.md). 
 
 > [!NOTE]
-> Streaming quality is highly dependent on the strength of your wifi network.
+> * Streaming quality is highly dependent on the strength of your wifi network.
+> * All capabilities are automatically enabled for the holographic remoting player. If you find a capability that requires user permission (ex: eye tracking) to be working over streaming but not when running on device, check to ensure you've enabled the proper capabilities under your project settings.
```
Comment (hferrone) [inline mixed-reality-docs/unreal-streaming.md:31]: Is there a link to how/what capabilities to enable for this?

File: mixed-reality-docs/unreal-deploying.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+## Overview
+There are two ways to deploy an Unreal application to HoloLens 2: 
+* Directly from the Unreal editor
+* As a package uploaded via the device portal
+
+Both options require you to set up your HoloLens to use the [device portal](using-the-windows-device-portal.md) for development. 
+
+## Deploying to device from the Unreal editor
+
+1. Click the dropdown arrow next to the "Launch" button. Initially the HoloLens device option will be grayed out.
```
Comment (hferrone) [inline mixed-reality-docs/unreal-deploying.md:25]: "Initially, the HoloLens device option will be grayed out."

Comment (hferrone) [inline mixed-reality-docs/unreal-deploying.md:25]: Bold "Launch"

File: mixed-reality-docs/unreal-deploying.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+![Launch dropdown options](images/unreal/launch-dropdown.png)
+
+2. Open the **Device Manager**.
+
+3. Your HoloLens will not appear in the device list automatically. Expand the **Add An Unlisted Device** section.
```
Comment (hferrone) [inline mixed-reality-docs/unreal-deploying.md:31]: won't

File: mixed-reality-docs/unreal-deploying.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+Both options require you to set up your HoloLens to use the [device portal](using-the-windows-device-portal.md) for development. 
+
+## Deploying to device from the Unreal editor
+
+1. Click the dropdown arrow next to the "Launch" button. Initially the HoloLens device option will be grayed out.
+
+![Launch dropdown options](images/unreal/launch-dropdown.png)
+
+2. Open the **Device Manager**.
```
Comment (hferrone) [inline mixed-reality-docs/unreal-deploying.md:29]: "Open the **Device Manager**. Note that your HoloLens won't automatically appear in the device list."

File: mixed-reality-docs/unreal-deploying.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+3. Your HoloLens will not appear in the device list automatically. Expand the **Add An Unlisted Device** section.
+
+4. Select "HoloLens" as your **Platform**.
```
Comment (hferrone) [inline mixed-reality-docs/unreal-deploying.md:33]: Bold HoloLens

File: mixed-reality-docs/unreal-deploying.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+5. Enter the IP address and port of the device as the device identifier, e.g. "127.0.0.1:10080" (when connected via USB). Use the same user name and password as used by the Device Portal.
```
Comment (hferrone) [inline mixed-reality-docs/unreal-deploying.md:35]: Enter your devices' IP address and port information separated by a colon as the device identifier. For example, "127.0.0.1:10080" (when connected via USB). Use your Device Portal username and password credentials.

File: mixed-reality-docs/unreal-deploying.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+2. Open the **Device Manager**.
+
+3. Your HoloLens will not appear in the device list automatically. Expand the **Add An Unlisted Device** section.
+
+4. Select "HoloLens" as your **Platform**.
+
+5. Enter the IP address and port of the device as the device identifier, e.g. "127.0.0.1:10080" (when connected via USB). Use the same user name and password as used by the Device Portal. 
+
+6. Hit **Add** and close the device manager. 
+    * In the case of an error (such as wrong address, user name or password), an error message will be printed to the Output Log.
```
Comment (hferrone) [inline mixed-reality-docs/unreal-deploying.md:38]: "a message will be..."

File: mixed-reality-docs/unreal-deploying.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+
+![Adding an unlisted device](images/unreal/add-unlisted-device.png)
+
+7. Click the dropdown arrow next to the "Launch" button again- this time you should see the HoloLens device you just added. Select the HoloLens device to build and deploy to your HoloLens.
```
Comment (hferrone) [inline mixed-reality-docs/unreal-deploying.md:42]: Bold Launch and add a space between "again" and "-"

File: mixed-reality-docs/unreal-streaming.md (hunk 1 above)
Comment (sw5813) [reply mixed-reality-docs/unreal-streaming.md:31]: I added this comment to explain that for remoting you wouldn't need to enable any capabilities (they're all enabled by default). Whether you would need any in your application depends on what your application does (ex: getting user consent for eye tracking). Does that make sense?

**Pull request merged**